react/react-training/src/redux/game/actions.js

```diff
@@ -0,0 +1,26 @@
+// noinspection JSUnusedGlobalSymbols
```

Comment:
Do you need this?

Reply:
Yesss... Or do you prefer I delete that comments?